### PR TITLE
Dockerize voice building for Android

### PR DESCRIPTION
**Pull request by lumpidu**
This creates the voice Álfur completey from scratch, in building it for 4 Android platforms by adding a voice loader driver layer to it. The result are 4 `.so` shared objects + accompanying header file which are extraced from the built container `lvl_is:latest` to the local directory `voice_deliverables/`. There is a meta-data file `voice_info.json` that can be parsed by a program to get hold of all platform files.

### CODE REVIEW

File: docker/Dockerfile.festvox [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+# Build Festival
+WORKDIR /usr/local/src/festival
+RUN ./configure && make
+
+# DS: ext/ehmm_patch, for clustergen voices
+RUN git clone https://github.com/lumpidu/unit-selection-festival.git && cp -r unit-selection-festival/ext /usr/local/src/ext
+
+# Build Festvox
+WORKDIR /usr/local/src/festvox
+RUN cp /usr/local/src/ext/ehmm_patch/do_ehmm ./src/ehmm/scripts/
```
Comment (G-Thor) [inline docker/Dockerfile.festvox]: The ehmm patch is no longer necessary if we use the modified phoneset

File: docker/Dockerfile.festvox [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+# Build the Edinburgh Speech Tools
+WORKDIR /usr/local/src/speech_tools
+RUN ./configure && make
+
+# Build Festival
+WORKDIR /usr/local/src/festival
+RUN ./configure && make
+
+# DS: ext/ehmm_patch, for clustergen voices
+RUN git clone https://github.com/lumpidu/unit-selection-festival.git && cp -r unit-selection-festival/ext /usr/local/src/ext
```
Comment (G-Thor) [inline docker/Dockerfile.festvox]: Since we don't need the ehmm patch, this is probably unnecessary as well.

**Review by G-Thor — APPROVED** [review]
This seems fine to me, although as I mention the ehmm patch is probably no longer necessary if we work around the issue by modifying the phoneset.

File: build_docker_lvl_is.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -euo pipefail
+docker build -f docker/Dockerfile.festvox -t harbour.grammatek.com/sim/festvox-build-tools:latest .
+docker build -f docker/Dockerfile.lvl_is --progress=plain -t harbour.grammatek.com/sim/lvl_is:latest .
```
Comment (G-Thor) [inline build_docker_lvl_is.sh:5]: Will Grammatek be hosting these images?

File: docker/Dockerfile.festvox (hunk 1 above)
Comment (lumpidu) [reply docker/Dockerfile.festvox]: ok, I'll try to build without the patch

File: docker/Dockerfile.festvox (hunk 2 above)
Comment (lumpidu) [reply docker/Dockerfile.festvox]: right, will change it accordingly

File: build_docker_lvl_is.sh (hunk 3 above)
Comment (lumpidu) [reply build_docker_lvl_is.sh:5]: this is our docker registry, which we have made publicly readable for the sim project. We are hosting these images there so one can just pull them without the need to build these locally

**Pull request merged**